### DISCUSSION TITLE
WiFi Direct File Transfer

### DISCUSSION THREAD

**Discussion by ghost** [spacedriveapp/spacedrive#2537] [Feature Request]
Is this possible? It's really great. No internet required, it's magical. It's like airdrop.

**Comment by fvnju** [comment]
Isn't it the spacedrop functionality that you can enable in settings? If it doesn't work for you, you can try localsend.
![settings](https://github.com/spacedriveapp/spacedrive/assets/147294177/11aa0e38-01a2-434b-8ad3-cbe5efc7e993)
![snapdrop](https://github.com/spacedriveapp/spacedrive/assets/147294177/b87b3625-6221-40c9-8070-e2561545d092)
